tests/include/catch_wrapper.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation

```

Comment:
Can you just use external\ebpf-for-windows\tests\libs\util\catch_wrapper.hpp instead of adding this file here?

Reply:
Done.